### PR TITLE
Html forms - Chris Tran

### PR DESCRIPTION
**Pull request by tran-christopher**
https://github.com/tran-christopher/c1023-code-solutions/assets/145728509/fec3db52-ffeb-4d84-bcb2-5b0a2ad83358

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    </p>
+    <p>
+      <label for="textarea">Comments</label>
+    </p>
+    <p>
+      <textarea id="textarea" name="textarea"></textarea>
+    </p>
+    <p>
+      <button type="submit">Submit</button>
+    </p>
```
Comment (shawnkost) [inline html-forms/index.html]: All of this needs to be wrapped inside of a `form` tag

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <p>
+      <label for="email">Email</label>
+      <input type="email" id="email" name="email" />
+    </p>
+    <p>
+      <label for="dropdown" id="dropdown">Why is coding cool?</label>
+      <select>
```
Comment (shawnkost) [inline html-forms/index.html]: The `id="dropdown"` needs to go on the `select`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Couple changes here Chris!

**Comment by tran-christopher** [comment]
Animation

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**